aries-site/src/layouts/main/Layout.js

```diff
@@ -24,51 +29,59 @@ export const Layout = ({ children, descriptiveHeader, title, isLanding }) => {
     <ThemeMode>
       <ResponsiveContext.Consumer>
         {size => (
-          <Box
-            height={{ min: '100vh' }}
-            margin="auto"
-            width={{ max: 'xxlarge' }}
-          >
-            <Head title={title} />
-            <Header
-              background={
-                descriptiveHeader && descriptiveHeader.props.background
-              }
-            />
-            <Main overflow="visible">
-              {/* Allows DescriptiveHeader background color not to be
-               * confined by formatting restrictions of page content
-               */}
-              {descriptiveHeader &&
-                React.cloneElement(descriptiveHeader, {
-                  pad: {
-                    horizontal: calcPad(size),
-                    bottom: calcPad(size),
-                    top: 'xlarge',
-                  },
-                  round: { corner: 'bottom', size: 'medium' },
-                })}
-              {/* aligns with responsive padding for aries-core Nav */}
-              <Box
-                pad={{
-                  horizontal: calcPad(size),
-                  bottom: calcPad(size),
-                  top: 'medium',
-                }}
-              >
-                {children}
-                {!isLanding && <FeedbackSection />}
-              </Box>
-            </Main>
-            <Footer />
-          </Box>
+          <Stack fill guidingChild={backgroundImage && 'last'}>
```

Comment:
can you explain `guidingChild={backgroundImage && 'last'}` ?  
if `backgroundImage` exists the guiding child is the last?

Reply:
On the pages when we have a backgroundImage, the page content becomes the `last` child in the Stack and we want that to drive the layout.

For details pages, we don't have a background image. So, the only child of the Stack would be the Box containing the page content (now the first child), in which case we would want that to guide. Therefore, this condition checks for that. We need this order of children within the Stack because otherwise the backgroundImage sits on top of the text.

Reply:
I can add a comment in the file.

Reply:
Added comment.